app/assets/javascripts/helpers/table_row_form_helpers.js

```diff
@@ -0,0 +1,48 @@
+//= require helpers/form_helpers
+
+var TABLE_ROW_FORM_HELPERS = (function ($, FORM_HELPERS){
+  'use strict';
+
+  /**
+   * Initializes elements that act as forms, but are not form nodes. In particular,
+   * this is used in the case where each row in a table acts as a form.
+   *
+   * @param {String} selector Selector for the form's submit button.
+   */
+  function initialize(selector) {
+    $(document).on('click', selector, onSubmit);
+  }
+
+  /**
+   * Handles the submit button click event.
+   *
+   * @param e The event object.
+   */
+  function onSubmit(e) {
```

Comment:
I configured jslint to skip such check for functions, not sure why it's still here. have you rebased ?